sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -393,4 +399,46 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       })
       .getOne();
   }
+
+  async assessOffering(
+    offeringId: number,
+    institutionId: number,
+    userId: number,
+    assessmentNotes: string,
+    offeringStatus: OfferingStatus,
+  ): Promise<void> {
+    return this.connection.transaction(async (transactionalEntityManager) => {
+      // create the note for assessment.
+      const user = {
+        id: userId,
+      } as User;
+      const note = new Note();
+      note.description = assessmentNotes;
+      note.noteType = NoteType.Program;
+      note.creator = user;
+      const noteEntity = await transactionalEntityManager
+        .getRepository(Note)
+        .save(note);
+
+      // update program
```

Comment:
full stop

Reply:
Actually the comment itself is wrong 😊 It was copied from approve program. so it came here as update program.